.travis.yml

```diff
@@ -16,11 +16,16 @@ python:
 node_js:
 - '0.10'
 addons:
+  artifacts:
+    paths:
+    - results
+  firefox: "latest-esr"
```

Comment:
I could see how this might affect Slimer. Basically, without this we have no control over the version of Firefox the Travis run has access to (it depends on when they update their containers). The Selenium tests need at least a supported version of Firefox, and Selenium support latest, latest-1, latest-esr, and latest-esr-1. I would hope that Slimer should have a similar support matrix, but perhaps not.